saleor/account/forms.py

```diff
@@ -120,10 +119,73 @@ def get_users(self, email):
         return active_users
 
     def send_mail(
-            self, subject_template_name, email_template_name, context,
-            from_email, to_email, html_email_template_name=None):
+            self,
+            subject_template_name,
+            email_template_name,
+            context,
+            from_email,
+            to_email,
+            html_email_template_name=None):
         # Passing the user object to the Celery task throws an
         # error "'User' is not JSON serializable". Since it's not used in our
         # template, we remove it from the context.
         del context['user']
         emails.send_password_reset_email.delay(context, to_email)
+
+
+class EmailChangeForm(forms.ModelForm):
+    """ Allow changing email """
```

Comment:
I think this docstring is not especially helpful.

It should be added without spaces around `"""` and end with a full stop.